keps/sig-node/4033-group-driver-detection-over-cri/README.md

```diff
@@ -265,231 +177,85 @@ will take precedence over cgroupDriver setting from the kubelet config (or
 `--cgroup-driver` command line flag). If the runtime does not provide
 information about the cgroup driver, then kubelet will fall back to using its
 own configuration (`cgroupDriver` from kubeletConfig or the `--cgroup-driver`
-flag). Further, the kubeletConfig field and `--cgroup-driver` flag will be
-marked as deprecated, to be dropped when support for the feature is adopted by
-CRI-O and containerd. Usage of the deprecated setting will produce a log
+flag). For beta, the kubeletConfig field and `--cgroup-driver` flag will be
+marked as deprecated. Usage of the deprecated setting will produce a log
 message, e.g.:
 
 ```
 cgroupDriver option has been deprecated and will be dropped in a future release. Please upgrade to a CRI implementation that supports cgroup-driver detection.
 ```
 
+The `--cgroup-driver` flag and the cgroupDriver configuration option will be
+deprecated and have no effect when support for the feature is graduated to GA.
```

Comment:
What's the motivation behind making CRI auto-detection a *hard* requirement and ignoring allowing explicit user configuration (which would enable using older runtimes for example?), the flag doesn't seem like a lot of code to retain.

Reply:
+1 same question. seems to add risk with little benefit?

Reply:
Now changed so that the fallback behavior is kept as long as the cmdline flag and config option. These will be marked as deprecated and will eventually be removed as per the K8s deprecation policy.